### PR TITLE
fix version

### PR DESCRIPTION
**Pull request by uzi18**
[me@here esp-link]$ git describe --tags --match 'v*.0' --long --dirty | sed -re 's/(\.0)?-/./'
v3.1.47-g9c6530d-dirty
[me@here esp-link]$ git describe --tags --match 'v*.*.*' --long --dirty | sed -re 's/(\.0)?-/./'
v3.2.47.alpha.0-g9c6530d-dirty

### CODE REVIEW

**Comment by uzi18** [comment]
fix for #348

File: Makefile [hunk 1]
```diff
@@ -191,7 +191,7 @@ TRAVIS_BRANCH?=$(shell git symbolic-ref --short HEAD --quiet)
 # Use git describe to get the latest version tag, commits since then, sha and dirty flag, this
 # results is something like "v1.2.0-13-ab6cedf-dirty"
 NO_TAG ?= "no-tag"
-VERSION := $(shell (git describe --tags --match 'v*.0' --long --dirty || echo $(NO_TAG)) | sed -re 's/(\.0)?-/./')
+VERSION := $(shell (git describe --tags --match 'v*.*.*' --long --dirty || echo $(NO_TAG)) | sed -re 's/(\.0)?-/./')
```
Comment (tve) [inline Makefile:194]: What is this fixing?

**Review by tve — COMMENTED** [review]
The console fix looks great and ready to merge, thanks!
I don't know what the VERSION thing does / breaks...

File: Makefile (hunk 1 above)
Comment (uzi18) [reply Makefile:194]: You have got it in commit:
[me@here esp-link]$ git describe --tags --match 'v*.0' --long --dirty | sed -re 's/(.0)?-/./'
v3.1.47-g9c6530d-dirty
[me@here esp-link]$ git describe --tags --match 'v*..' --long --dirty | sed -re 's/(.0)?-/./'
v3.2.47.alpha.0-g9c6530d-dirty

fix needed as it takes wrong version ... into VERSION variable. try it.

**Comment by uzi18** [comment]
one more fix to integrate

File: Makefile (hunk 1 above)
Comment (uzi18) [reply Makefile:194]: @tve is it clear now?

**Comment by uzi18** [comment]
You want me to split this PR for 3 different PRs?

**Comment by uzi18** [comment]
@tve ready to merge

**Comment by tve** [comment]
Thanks! I'm heading out on vacation and am scrambling... I'll try to take a look in the next couple of days. So from your point of view it's tested?

**Comment by uzi18** [comment]
Just check above listed issues confirmed as "works for me" by users.

**Comment by tve** [comment]
Thanks for putting all this together!

**Pull request merged**